### PR TITLE
Fixing an ansible-lint warning "wrong indentation: expected 8 but fou…

### PR DESCRIPTION
**Pull request by nhosoi**
…nd 6  (indentation)"

### CODE REVIEW

File: tests/tests_files_elasticsearch_certs_incomplete.yml [hunk 1]
```diff
@@ -21,8 +21,8 @@
         - "{{ __test_cert }}"
         - "{{ __test_key }}"
 
-    - block:
-      - name: deploy config to send to elasticsearch
+    - name: deploy config to send to elasticsearch
+      block:
```
Comment (pcahyna) [inline tests/tests_files_elasticsearch_certs_incomplete.yml:25]: This does not look correct. You essentially deleted the list item with the `include_role` below.

File: tests/tests_files_elasticsearch_certs_incomplete.yml [hunk 2]
```diff
@@ -47,7 +47,6 @@
         include_role:
           name: linux-system-roles.logging
 
-      - name: unreachable task
```
Comment (pcahyna) [inline tests/tests_files_elasticsearch_certs_incomplete.yml:50]: Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.

Comment (nhosoi) [reply tests/tests_files_elasticsearch_certs_incomplete.yml:50]: > Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.

Yeah, obviously this patch is broken :( Sorry for pushing it too early.

Comment (nhosoi) [reply tests/tests_files_elasticsearch_certs_incomplete.yml:50]: > > Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.
> 
> Yeah, obviously this patch is broken :( Sorry for pushing it too early.

@pcahyna, I've updated the patch. All I had to do was adding 2 column indentation... :p

**Comment by nhosoi** [comment]
Thank you for the reviews and approval, @pcahyna and @richm. Merging.

**Pull request merged**